src/Main.java

```diff
@@ -1,7 +1,68 @@
+import java.time.LocalDate;
+
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
+
     public static void main(String[] args) {
 
+        System.out.println("");
+        System.out.println("Задание N1");
+
+        printIsYearLeap(2000);
+
+        System.out.println("");
+        System.out.println("Задание N2");
+
+        printAppVersion (2, 2024);
+
+        System.out.println("");
+        System.out.println("Задание N3");
+
+        int deliveryDays =calculateTimePeriod(800);
+        if (deliveryDays != -1) {
+            System.out.println("Вашу карту доставят за " + deliveryDays);
+        } else {
+            System.out.println("Доставка карты на такое расстояние невозможна.");
+        }
+    }
+
+    public static void printIsYearLeap(int year) {
+
+        if (year < 1584) {
+            System.out.println("До 1584 года понятия високосного года не существовало.");
+        } else if (year > 1584 && year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```

Comment:
```suggestion
        } else if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```
условие лишнее, т.к. оно уже проверяется в if, а если else if, значит противоположно тому, т.е. `year >= 1584`